src/main/java/org/jmxtrans/embedded/config/EtcdKVStore.java

```diff
@@ -219,7 +219,7 @@ public KeyValue getKeyValue(String KeyURI) throws EmbeddedJmxTransException {
   }
 
   private URL[] makeEtcdBaseUris(String etcdURI) throws EmbeddedJmxTransException {
-    String serverList = null;
+    String serverList;
```

Comment:
This is probably a good candidate for a method extraction and the removal of the conditional assignment. This would IMHO bring more clarity than just removing the dead store.